pkg/scheduler/plugins/elasticquota/pod_handler_test.go

```diff
@@ -172,3 +173,45 @@ func TestPlugin_OnPodUpdateWhenDisableDefaultAndRootTree(t *testing.T) {
 	assert.True(t, quotav1.Equals(test1QuotaInfo.GetUsed(), MakeResourceList().CPU(0).Mem(0).Obj()))
 	assert.True(t, quotav1.Equals(test2QuotaInfo.GetUsed(), MakeResourceList().CPU(0).Mem(0).Obj()))
 }
+
+func TestPlugin_OnPodDeleteParseObj(t *testing.T) {
+	defer utilfeature.SetFeatureGateDuringTest(t, k8sfeature.DefaultMutableFeatureGate, koordfeatures.DisableDefaultQuota, true)()
+
```

Comment:
This UT contains multiple Cases. It is recommended to use tabel-driven form to write this case, similar to the following:
```go
func TestPlugin_OnPodDelete(t *testing.T) {
	tests := []struct {
		name string
        // here add args fields.
	}{}
	for _, tt := range tests {
		t.Run(tt.name, func(t *testing.T) {})
	}
}
```